index.html

```diff
@@ -26,8 +26,125 @@ <h1 class="detail one">I'm Mariane<br>Glad to see<br>you</h1>
-</main>
+     </section>
+    </div>
+  
+    <div class="cards">
+      <div class="card">
+        <img src="image/Snapshoot-Portfolio.png" alt="A photo of a website page">
+        <div class="card-content">
+          <div class="card-heading">Tonic</div>
+          <ul class="tool1">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>        
+          <div class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</div>
+          <ul class="tool2">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+          </ul>
+          <a href="#" class="detail-style2">See Project</a>
+        </div>
+    </div>
+    <div class="card">
+      <img src="image/Snapshoot-Portfolio-(1).png" alt="A photo of a website page">
+      <div class="card-content">
+        <div class="card-heading">Tonic</div>
+        <ul class="tool1">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>        
+        <div class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</div>
+        <ul class="tool2">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javascript</a></li>
+        </ul>
+        <a href="#" class="detail-style2">See Project</a>
+      </div>
+    </div>
+    <div class="card">
+      <img src="image/Snapshoot-Portfolio-(2).png" alt="A photo of a website page">
+      <div class="card-content">
+        <div class="card-heading">Tonic</div>
+        <ul class="tool1">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>        
+        <div class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</div>
+        <ul class="tool2">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javascript</a></li>
+        </ul>
+        <a href="#" class="detail-style2">See Project</a>
+      </div>
+    </div>
+    <div class="card">
+      <img src="image/Snapshoot-Portfolio-(3).png" alt="A photo of a website page">
+      <div class="card-content">
+        <div class="card-heading">Tonic</div>
+        <ul class="tool1">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>        
+        <div class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</div>
+        <ul class="tool2">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javascript</a></li>
+        </ul>
+        <a href="#" class="detail-style2">See Project</a>
+      </div>
+    </div>
+    </div>
+
+    <div class="about-section">
+      <div class="myself">
+        <h1>About<br>Myself</h1>
```

Comment:
- [ ] Please make sure your web app only has on `h1` tag, all remaining must be `hx` bellow, since you have already used an h1 for your intro title, please downgrade the rest accordingly.